packages/ir-sdk/fern/apis/ir-types-latest/definition/ir.yml

```diff
@@ -55,6 +56,7 @@ types:
       variables: list<variables.VariableDeclaration>
       serviceTypeReferenceInfo: ServiceTypeReferenceInfo
       readmeConfig: optional<ReadmeConfig>
+      sourceConfig: optional<SourceConfig>
```

Comment:
I think the reality is that the `source` can be multiple OpenAPI specs, FernDefinitions or Proto files. Thoughts on making this: 
```yaml
sources: 
  type: list<ApiDefinitionSource>
  docs: The raw API definitions that produced the IR 
  
ApiDefinitionSource: 
  union: 
    proto: ProtoSource
    openapi: OpenAPISource # can skip for now
```

Reply:
The C# generator can just handle processing a single (the first) `ProtoSource` for now

Reply:
Also would be great if each source had a unique `SourceId`

Reply:
Nice call - the `ProtoSourceConfig` as-is handles the multiple `.proto` files (via the root directory URL reference), but it doesn't include mixing and matching between Fern / OpenAPI / Proto.

I wonder if it's worth still wrapping `sources` in an object so it's easy to extend later. Thoughts on this mild adjustment?

```yaml
SourceConfig:
  properties: 
    sources:
      docs: The raw API definitions that produced the IR.
      type: list<ApiDefinitionSource>
  
ApiDefinitionSource:
  union:
    proto: ProtoSource
    openapi: {}
```